index.html

```diff
@@ -44,7 +44,149 @@ <h5> I can help you build a product ,
+           <h3>Profesional Art <br>
+                Printing Data</h3>
+           <p>A daily selection of privately 
+               personalized reads; no accounts 
+               or sign-ups required. has been 
+               the industry's standard</p>
+           <div class="language container"> 
+               <span class="html">html</span> 
+               <span class="bootstrap boot1">bootstrap</span>
+               <span class="ruby ">ruby</span>
+               </div>
+           <button type="button" class="see-project"> See Project</button>
+         </div>
+         <div class="profesional p-4">
+           <h3>Profesional Art <br>
+                Printing Data</h3>
+           <p>A daily selection of privately 
+               personalized reads; no accounts 
+               or sign-ups required. has been 
+               the industry's standard</p>
+           <div class="language container"> 
+               <span class="html">html</span> 
+               <span class="bootstrap boot1">bootstrap</span>
+               <span class="ruby ">ruby</span>
+               </div>
+           <button type="button" class="see-project"> See Project</button>
+         </div>
+         <div class="profesional p-5">
+           <h3>Profesional Art <br>
+                Printing Data</h3>
+           <p>A daily selection of privately 
+               personalized reads; no accounts 
+               or sign-ups required. has been 
+               the industry's standard</p>
+           <div class="language container"> 
+               <span class="html">html</span> 
+               <span class="bootstrap boot1">bootstrap</span>
+               <span class="ruby ">ruby</span>
+               </div>
+           <button type="button" class="see-project"> See Project</button>
+         </div>
+         <div class="profesional p-6">
+           <h3>Profesional Art <br>
+                Printing Data</h3>
+           <p>A daily selection of privately 
+               personalized reads; no accounts 
+               or sign-ups required. has been 
+               the industry's standard</p>
+           <div class="language container"> 
+               <span class="html">html</span> 
+               <span class="bootstrap boot1">bootstrap</span>
+               <span class="ruby ">ruby</span>
+               </div>
+           <button type="button" class="see-project"> See Project</button>
+         </div>
+   </section>
+   <section class="about">
+       <h1> About me</h1>
+       <p>Hello I’m a software developer!
+            I can help you build a product , feature
+             or website Look through some of my work 
+            and experience! If you like what you
+            see and have a project you need coded, 
+             don’t hestiate to  contact me.</p> 
+       
+             <button type="button" class="get">Get My Resume</button>
+             <img class="about-img" src="images/Group1.png" alt="">
```

Comment:
- [ ] The button (e.g., "Get My Resume") should follow the button styling: Active, Hover, and Pressed (see design below), and please implement it
### Figma design
![fimga](https://user-images.githubusercontent.com/81727988/193013682-4b947d62-c543-4a72-bcba-1bd52f9f5e5b.png)